lib/setup.js

```diff
@@ -65,7 +65,9 @@ function registerPartials(items) {
   // Compile all the loaded partials
   _.forIn(hbs.partials, function (value, key) {
     if (_.isString(value)) {
-      hbs.partials[key] = hbs.compile(value);
+      //preventIndent:true solve issue with  <pre> tag/white-space: pre
```

Comment:
Seems like circleci is failing because of the linter. Just a whitespace after the `//`
```suggestion
      // preventIndent:true solve issue with  <pre> tag/white-space: pre
```

Reply:
done